tests/messages/test_frontend.py

```diff
@@ -761,6 +761,19 @@ def test_usage(self):
 pybabel: error: no valid command or option passed. try the -h/--help option for more information.
 """, sys.stderr.getvalue().lower())
 
+    def test_list_locales(self):
+        """
+        Test the command with the --list-locales arg.
+        """
+        result = self.cli.run(sys.argv + ['--list-locales'])
+        self.assertEqual(0, result)
```

Comment:
Prefer regular assertions in new tests:
```suggestion
        assert result == 0
```